weave-js/src/components/PagePanelComponents/Home/Browse3/inference/ModelDetailsLoaded.tsx

```diff
@@ -0,0 +1,331 @@
+import copyToClipboard from 'copy-to-clipboard';
+import React, {useCallback, useState} from 'react';
+import {useHistory} from 'react-router-dom';
+import {toast} from 'react-toastify';
+
+import {TargetBlank} from '../../../../../common/util/links';
+import {Button} from '../../../../Button';
+import {CodeEditor} from '../../../../CodeEditor';
+import {Icon, IconPrivacyOpen} from '../../../../Icon';
+import {ToggleButtonGroup} from '../../../../ToggleButtonGroup';
+import {Tooltip} from '../../../../Tooltip';
+import {Link} from '../pages/common/Links';
+import {DetailTile} from './DetailTile';
+import logoHuggingFace from './logos/icon-hugging-face.png';
+import {Modalities} from './Modalities';
+import {navigateToPlayground} from './navigate';
+import {InferenceContextType, Model} from './types';
+import {
+  getContextWindowString,
+  getLaunchDateString,
+  getModelLabel,
+  getModelLicense,
+  getModelLogo,
+  getModelSourceName,
+  getPriceString,
+  getShortNumberString,
+  INFERENCE_PATH,
+} from './util';
+
+type ModelDetailsLoadedProps = {
+  model: Model;
+  inferenceContext: InferenceContextType;
+};
+
+const BASE_URL = 'https://infr.cw4637-staging.coreweave.app/v1';
+
+// # Enable HTTPX debugging
+// import logging
+// logging.basicConfig(level=logging.DEBUG)
+
+// TODO: Should probably pull this out into a central place
+const CODE_EXAMPLES_CHAT: Record<string, string> = {
+  Python: `
+import openai
+import weave
+
+# Set a custom base URL
+client = openai.OpenAI(
+    base_url='${BASE_URL}',
+    # Generally recommend setting OPENAI_API_KEY in the environment
+    api_key="some-key-value"
```

Comment:
at some point anyone who might remotely stumble upon this and derive value from it should know what to do here anyway